docs/docusaurus/docs/changelog.md

```diff
@@ -2,6 +2,14 @@
 title: Changelog
 ---
 
+### Deprecation policy
```

Comment:
Nit - Should "policy" be "Policy"?

Not sure what the convention is so I'll defer to youj

Reply:
The convention is to only capitalize the first word of a header and any technical terms within the header.